fractions/jaeger/src/main/resources/modules/com/uber/jaeger/main/module.xml

```diff
@@ -0,0 +1,18 @@
+<module xmlns="urn:jboss:module:1.3" name="com.uber.jaeger">
+  <resources>
+    <artifact name="com.uber.jaeger:jaeger-core:${version.jaeger}"/>
+    <artifact name="com.uber.jaeger:jaeger-tracerresolver:${version.jaeger}"/>
+    <artifact name="com.uber.jaeger:jaeger-thrift:${version.jaeger}"/>
+    <artifact name="org.apache.thrift:libthrift:0.9.2"/>
```

Comment:
I'd investigate a few things here:

* Does WildFly have any of these artifacts already as compatible modules (commons-logging? commons-codec?)  If so, simply reference them the same was as you have `org.slf4j`.

* If WildFly does *not* have compatible versions of these modules, I'd recommend creating a new `module.xml` for each logical portion, possibly setting the `slot` to be something other than `main` as it might conflict with the `main` module coming from WildFly.  See our Ribbon bits for how, for instance, we brought in different versions of things like `guava` so as to not conflict with the core WildFly module, while also still keeping the `module.xml` as usual.

Additionally, *never* include versions direction within the `module.xml`, but prefer the `${version.whatever}` notations, using properties from the root `pom.xml`.  If required, add grist to the property name to include `jaeger` in case there are 3 other versions of `commons-codec` in use and defined via properties. in the `pom.xml`.